<launch>

  <group ns="iauv1">
  <node name="nav_sim" package="vehicle_core" type="node_nav_sim.py" />
  <node name="pilot" package="vehicle_core" type="node_pilot.py" />
  <node name="dxm" package="mango_dxm" type="dxm.py" args="2" />
  <node name="exec" package="mango_dxm" type="iauv_exec.py" args="2" />
  </group>

</launch>
